docs/content/guides/developer/sui-101/avoid-equivocation.mdx

```diff
@@ -4,29 +4,45 @@ description: Avoid unintentionally locking objects with your smart contract logi
 
-The other source of equivocation does not originate from bad intentions. There are logic traps that smart contract code can inadvertently trigger, like when dealing with [sponsored transactions](https://docs.sui.io/guides/developer/sui-101/sponsor-txn#client-equivocation-risk). The effects of this type of equivocation can lock the objects your code interacts with until the end of the current epoch. This can lead to frustration for you, and more importantly, your users.
+Object versioning tracks the state of objects across transactions and epochs. 
 
-To avoid double spending, validators lock objects as they validate transactions. An equivocation occurs when an owned object pair (`ObjectId`, `SequenceNumber`) is concurrently used in multiple non-finalized transactions.
+## What is equivocation?
 
-Perhaps the most common source of equivocation stems from the attempt to pay for gas using the same coin object. This can be a problem when performing multiple transactions that originate from the same sender.
+*Equivocation* occurs when an owned object pair (`ObjectId`, `SequenceNumber`) is used concurrently in multiple non-finalized transactions. Any object used in multiple transactions can become a source of equivocation if not managed correctly. 
 
-It's not just objects that can cause equivocation. Any object used in multiple transactions can be a source of equivocation if not handled properly. Sui uses object versioning to track the status of objects across transactions and epochs. If a transaction modifies an object, then one of the results of that transaction is to update the version of that same object.
+Equivocation can occur when:
 
-The versioning architecture is what supports paying the gas for a series of transactions using the same gas coin. Among other processing, the first transaction advances the version number of the coin and returns it to the sender. The next transaction can then use that same coin to pay for its gas. Equivocation is avoided because each transaction references a different version of the same coin.
+- A user or smart contract submits two transactions using the same (`ObjectId`, `SequenceNumber`) before the first is finalized.
+
+- Inadvertent logic traps in smart contract code, for example, when handling [sponsored transactions](/guides/developer/sui-101/sponsor-txn#client-equivocation-risk).
+
+Even though both transactions will not be finalized, submitting equivocated transactions still causes other problems for both developers and users. For example, when equivocation is detected, the involved object(s) are locked until the end of the current epoch. Neither transaction can proceed, and the object(s) cannot be used in any other transaction. Bad actors could intentionally submit equivocated transactions to lock up important objects and intentionally disrupt dApps.
```

Comment:
```suggestion
Even though both transactions aren't finalized, submitting equivocated transactions still causes other problems for both developers and users. For example, when equivocation is detected, the involved objects are locked until the end of the current epoch. Neither transaction can proceed, and the objects cannot be used in any other transaction. Bad actors could intentionally submit equivocated transactions to lock up important objects and intentionally disrupt dApps.
```
Was never a fan of the `(s)` construct but I don't think it's in the style guide.